api/src/main/java/eu/coatrack/api/Report.java

```diff
@@ -0,0 +1,77 @@
+package eu.coatrack.api;
+
+import java.util.Date;
+
+public class Report {
+    private boolean isOnlyPaidCalls;
+    private boolean isForConsumer;
+    private Date from;
+    private Date until;
+    private ServiceApi selectedService;
+    private User selectedConsumer;
+
+    public Report(boolean isOnlyPaidCalls, boolean isForConsumer, Date from, Date until, ServiceApi selectedService, User selectedConsumer) {
+        this.isOnlyPaidCalls = isOnlyPaidCalls;
+        this.isForConsumer = isForConsumer;
+        this.from = from;
+        this.until = until;
+        this.selectedService = selectedService;
+        this.selectedConsumer = selectedConsumer;
+    }
+
+    public Report() {}
```

Comment:
Can be removed using `@NoArgsConstructor` on class level. Does it still work if you use protected instead? I guess this constructor is only used by Spring and not by developers.